src/app/docs/button/button-docs.component.html

```diff
@@ -197,7 +229,7 @@
             Use singular or plural direct objects to indicate whether the button acts on one or multiple items.
           </li>
           <li>
-            Don't include articles. For example, use <strong>Add constituent</strong> rather than <strong>Add a constituent</strong>.
+            Don't include articles. For example, use <strong>Delete constituent</strong> rather than <strong>Delete a constituent</strong>.
           </li>
           <li>
             Don’t append extra words such as “details.” For example, use <strong>Edit constituent</strong> rather than <strong>Edit constituent details</strong>.
```

Comment:
```suggestion
            Don’t append extra words such as “details.” For example, use <strong>Edit goal</strong> rather than <strong>Edit goal details</strong>.
```
Same suggestion to avoid "constituent" terminology.